Terminal.Gui/Views/TextField.cs

```diff
@@ -264,10 +269,9 @@ void Adjust ()
 			if (point < first) {
 				first = point;
 			} else if (first + point - (Frame.Width + offB) == 0 ||
-				  DisplaySize (text, first, point) >= Frame.Width + offB) {
+				  DisplaySize (text, first, point).Item1 >= Frame.Width + offB) {
```

Comment:
Should be `DisplaySize (...).Length` in order for the code to be more readable.

Reply:
@tig it's not possible to use the named items. It's only possible when we instantiate a new one. Please review that.